chips/stm32f4xx/src/lib.rs

```diff
@@ -29,6 +29,8 @@ pub mod tim2;
 pub mod trng;
 pub mod usart;
 
+pub mod pwr;
```

Comment:
Can you move the stm changes to a different PR? Or are these not supposed to be here?

Reply:
The user manual that I used specified that the PWR registers applied for the whole STM32F4xx family. That being said, I did not have the opportunity to test it on any other boards than the nucleo-stm32f429zi.

[ST_Processor_Manual.pdf](https://github.com/tock/tock/files/12214393/ST_Processor_Manual.pdf)

Reply:
I thought this PR was just for the RPi Pico RTC?

Reply:
This PR also includes RTC support for the nucleo_f429zi board (edited the chips folder for stm32f429 and the board foler for nucleo_f429zi). The code for RPi Pico RTC was mostly rebased from an older commit, while the code for the nucleo board was written by me.
In any case, what should I do with the PWR registers? The code is generic for all the STM32F42xxx and STM32F43xxx boards, so it would be a loss to just use it for the specific nucleo_f429zi board.